lms/tasks/celery.py

```diff
@@ -0,0 +1,109 @@
+"""Celery app and configuration."""
```

Comment:
This file is pretty much copy pasted from checkmate.

I haven't seen any value that would need to be different in LMS from the start.